src/openads-appnexus/AppNexusConnector.js

```diff
@@ -109,6 +108,8 @@ export default class AppNexusConnector {
         )
         if (normalizedInputs.adUnits.length > 0 && this._prebidClient) {
           this._prebidClient.addAdUnits({adUnits: normalizedInputs.adUnits})
+          if (this._prebidConfig)
```

Comment:
this will update the config on each debounce emission, why it's not in the line it was before, just when setting the config?

Reply:
The documentation doesn't speak about the order to call the setConfig method, that's why I figured I could call the method in the constructor, prior to the other calls, but it didn't work.
In the examples, the methods are always called in the same order: addAdUnits - setConfig - requestBids.
After doing tests, I came to the conclusion that it was the correct order to call the methods.

Reply:
👍 let's check it another day so, I'm sure that the order should be setConfig-addUnits-whatever